### PR TITLE
Add option to mark an item as missing, refs UIIN-419

### PR DESCRIPTION
**Pull request by aditya-matukumalli**

### CODE REVIEW

File: src/ViewItem.js [hunk 1]
```diff
@@ -240,9 +247,33 @@ class ViewItem extends React.Component {
     this.props.updateLocation({ layer: 'copyItem' });
   }
 
+  handleConfirm = (item, requestRecords) => {
+    const newItem = _.cloneDeep(item);
+    _.set(newItem, ['status', 'name'], 'Missing');
+
+    const newRequestRecord = _.cloneDeep(requestRecords);
```
Comment (mkuklis) [inline src/ViewItem.js]: @aditya-matukumalli can we just clone `requestRecords[0]` here?

Comment (aditya-matukumalli) [reply src/ViewItem.js]: @mkuklis Good catch. Will make the change

File: package.json [hunk 2]
```diff
@@ -300,6 +300,7 @@
     "sinon": "^7.0.0"
   },
   "dependencies": {
+    "@folio/react-intl-safe-html": "^1.0.200010",
```
Comment (zburke) [inline package.json]: This version does not exist outside CI. If there's a bug fix in react-intl-safe-intl we need to depend on, let's get it released and depend on a proper, released version here.

Comment (zburke) [reply package.json]: Actually, NBD, but dealt with in #514 / [UIIN-500](https://issues.folio.org/browse/UIIN-500)

**Pull request merged**